### PR TITLE
Project filters

### PR DESCRIPTION
**Pull request by zkhin**
Fixes: https://github.com/SeedCompany/cord-api-v3/issues/643

### CODE REVIEW

File: src/components/project/query.helpers.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import { inArray, node, Query, relation } from 'cypher-query-builder';
+import { ProjectFilters } from './dto';
+
+export function projectListFilter(query: Query, filter: ProjectFilters) {
+  query
+    .match([...propMatch('status')])
+    .call((q) =>
+      filter.status ? q.where({ status: { value: inArray(filter.status) } }) : q
+    )
+    .match([...propMatch('sensitivity')])
```
Comment (CarsonF) [inline src/components/project/query.helpers.ts:10]: We only want to match these two if they are given

Comment (zkhin) [reply src/components/project/query.helpers.ts:10]: You mean, I have to change `match` to `optionalMatch` for these two fields?

Comment (CarsonF) [reply src/components/project/query.helpers.ts:10]: I mean don't call match if the value is nil

Comment (zkhin) [reply src/components/project/query.helpers.ts:10]: Okay

**Review by CarsonF — APPROVED** [review]

**Pull request merged**